include/exercise_functions.php

```diff
@@ -0,0 +1,504 @@
+<?php
+
+function addExercise(){
+
+  $exerciseTypeId = getExerciseTypeId();
+
+  $exerciseName = $_REQUEST["exerciseName"];
+  $session = $_REQUEST["session"];
+  $exerciseTarget = $_REQUEST["target"];
+  $userId = $_SESSION['userId'];
+
+  debugOutput($_REQUEST);
+
+  db_Query("
+    INSERT INTO exercise(exerciseName, typeId, session, target, userId)
+    VALUES(:exerciseName, :typeId, :session, :target, $userId)
```

Comment:
unsure why you don't do `:userId` and then has `userId` in your parameters array like the other ones